enzyme.config.js

```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
```

Comment:
Here's your Enzyme configuration, so if this is working, you shouldn't need the configuration at all at the top of your test file.

Reply:
Also, newline!